Assets/Resources/JSON/GiftItems.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
All JSON files should have a corresponding spreadsheet in the design folder and be exported using this tool: https://chrome.google.com/webstore/detail/export-sheet-data/bfdcopkbamihhchdnjghdknibmcnfplk?hl=en (allow for easy updating)